woocommerce/woocommerce-gutenberg-products-block
# Reviews by product: Update review styling and content

This PR takes care of styling (to match the latest figma files), as well as adding the 'verified' icon for verified reviews, and implementing read-more functionality.

Part of #658

### Screenshots

![Reviews – local wordpress test 2019-08-06 15-12-05](https://user-images.githubusercontent.com/90977/62547308-ee897a80-b85c-11e9-9bf9-064fbc2d5cb9.png)

### How to test the changes in this Pull Request:

1. Add reviews by product block
2. Check styling in editor/on frontend
3. Buy a product. Review it. Check you have an icon besides your avatar.
4. Leave a long review (> 10 lines). Test read more link.
